### ISSUE TITLE
Opening a discussion from notification bar in multi discussion window

### ISSUE THREAD

**Issue by ollb** [muffinmad/pidgin-im-gnome-shell-extension#22]
When I have one pidgin window with several discussions, opening a discussion from notification bar focus the window but doesn't open the right discussion.

**Comment by muffinmad** [comment]
Yes, it's pidgin dbus command purple_conversation_present behavior. Can't help with it on extension side.

**Comment by muffinmad** [comment]
https://developer.pidgin.im/ticket/15020
